### PR TITLE
Move logout and add links.

### PR DESCRIPTION
**Pull request by tobwiens**
Move the logout button to the right. Add links to the other portals.

### CODE REVIEW

**Comment by tobwiens** [comment]
@ow2-proactive/developers: Two reviews please, so we can add this into the release.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java [hunk 1]
```diff
@@ -370,18 +371,49 @@ public void onClick(ClickEvent event) {
...
 
         return tools;
     }
 
+    private ToolStripButton getSchedulerLinkButton() {
+        ToolStripButton schedulerButton = new ToolStripButton("Scheduler");
+
+        schedulerButton.addClickHandler(new ClickHandler() {
+            public void onClick(ClickEvent event) {
+                Window.open("/rm", "", "");
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: I think there is a mistake, it should be "/scheduler", no ?

Comment (tobwiens) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMPage.java]: YES! you are right, thanks!

**Comment by tobwiens** [comment]
Thanks, I will merge.

**Pull request merged**